CMakeLists.txt

```diff
@@ -72,6 +72,7 @@ if(ENABLE_SWIFT)
                            ${SWIFT_RUNTIME_LIBDIR}/${CMAKE_SHARED_LIBRARY_PREFIX}swiftSwiftOnoneSupport${CMAKE_SHARED_LIBRARY_SUFFIX})
 
   set(INSTALL_TARGET_DIR "${INSTALL_LIBDIR}/swift/${SWIFT_OS}" CACHE PATH "Path where the libraries will be installed")
+  set(INSTALL_STATIC_TARGET_DIR "${INSTALL_LIBDIR}/swift_static/${SWIFT_OS}" CACHE PATH "Path where the static libraries will be installed")
```

Comment:
To be honest, I'm not sure I understand the point of these two cached variables.  They are part of the resource dir naming scheme for the compiler, so why not always compose them when used?

Reply:
I am happy to drop them